nestkernel/simulation_manager.cpp

```diff
@@ -813,27 +842,6 @@ nest::SimulationManager::update_()
 }
```

Comment:
Dear @apeyser and @wschenck, sorry for the late reply. The update of the synaptic elements can be done in any irregular fraction of simulation time. This is not relevant because the structural changes actually take place on a much larger time scale, so the accumulation of these small changes in the synaptic elements can be achieved at partial steps and the final results should be the same. 
To verify this I ran the simple structural plasticity examples and they provide the same output using the new Prepare() - Run() - Cleanup() functions as with Simulate()